pkg/command/eventing/eventing_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+package eventing
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestNewEventingCmd(t *testing.T) {
+	cmd := NewEventingCmd(nil)
+	assert.Check(t, cmd.HasSubCommands(), "cmd eventing should have subcommands")
+
+	// _, _, err := cmd.Find([]string{"prepare"})
```

Comment:
Same as before, remove these empty tests and create issue with a TODO list.

If you want to keep this code then move it to a branch. Use github for what’s it good for, organizing your code :)